docs/content/Release-Notes/0-31.mdx

```diff
@@ -0,0 +1,122 @@
+- The `shown` property now hides cubes/views from the `/meta` endpoint when set
+  to `true`
+
+> Be sure to check the
+> [Deprecation document](https://github.com/cube-js/cube.js/blob/master/DEPRECATION.md)
+> for more details on the above.
+
+## Multiple Data Sources
+
+Cube has had support for multiple data sources for quite some time now, but
+we’ve made it even easier to configure them now with environment variables.
+[Read the documentation](https://cube.dev/docs/config/multiple-data-sources) to
+see how it works.
+
+## Modeling in YAML
+
+We’ve often heard users they want the ability to author data models in languages
+other than JavaScript - specifically YAML and Python. Today, we’re excited to
+release a public preview for YAML data modeling in Cube. Check out the
+[introductory blog post](https://cube.dev/blog/introducing-cube-support-for-yaml-data-modeling),
+or dive straight into
+[the documentation](https://cube.dev/docs/schema/getting-started/yaml).
+
+## Views
+
+Designed as a layer over cubes, views act as a façade of your whole data model.
+They are a great tool for to defining metrics, managing governance and data
+access, and controlling ambiguous join paths. Learn more about them in the
+[introductory blog post](https://cube.dev/blog/introducing-views), read more
+about them [in the documentation](https://cube.dev/docs/schema/reference/view)
+or watch our in-depth
+[webinar on the topic](https://cube.dev/events/using-cube-views-for-metrics-management).
+
+## Lambda Pre-Aggregations
+
+Lambda pre-aggregations allow combining batched/processed data with fresh data
+at query time, this is super useful for real-time data applications. You can
+even take it a step further and combine batched data from a data warehouse with
+streaming data from ksqlDB; find more information
+[in the documentation](https://cube.dev/docs/caching/pre-aggregations/lambda-pre-aggregations).
+
+## Drivers
+
+We’re really excited to present two new drivers,
+[Firebolt](https://cube.dev/docs/config/databases/firebolt) and Trino. And to
+top it off, we’ve also added a brand-new driver for
+[ksqlDB](https://cube.dev/docs/config/databases/ksqldb).
+
+## SQL API
+
+The Cube SQL API now speaks the Postgres protocol instead of MySQL; the Postgres
+protocol is better supported and allows us to extend supports to dozens of
+downstream tools. We’ve also made a number of improvements relating syntax and
+operator support, and we’ve documented
+[any difference in behaviour](https://cube.dev/docs/backend/sql/reference/functions-operators)
+from regular SQL.
+
+## Downstream Tools
+
+We’ve been busy since the last release adding support for downstream tools. Cube
+now connects to:
+
+- [Appsmith](https://cube.dev/docs/config/downstream/appsmith)
+- [Tableau](https://cube.dev/docs/config/downstream/tableau/)
+- [Microsoft PowerBI](https://cube.dev/docs/config/downstream/powerbi/)
+- Thoughtspot
+- Amazon QuickSight
+- Sigma Computing
```

Comment:
@keydunov We don't currently have pages in our documentation for these

Reply:
That is fine